packages/postgres-database/src/simcore_postgres_database/models/groups.py

```diff
@@ -76,14 +110,22 @@ class GroupType(enum.Enum):
         server_default=sa.text(
             '\'{"read": true, "write": false, "delete": false}\'::jsonb'
         ),
+        doc="Group's access-rights to R/W/D a resource",
```

Comment:
```suggestion
        doc="User's access-rights to R/W/D a group",
```

Reply:
Aren't these access rights assigned to the group?

Reply:
hmm as far as I know this is the user_to_group table and defines what rights a user has in a group

Reply:
aha! I completely missed these access-rights!
what does read/write a group means in this context? is about creating/admin groups? 
I checked the db and saw that they are all set to ``{"read": true, "write": false, "delete": false}`` 
Do we really use this feature?